src/gd_kotlin.cpp

```diff
@@ -547,3 +547,20 @@ bool GDKotlin::initialized() const {
 const Vector<Pair<String, String>>& GDKotlin::get_configuration_errors() const {
     return configuration_errors;
 }
+
+void GDKotlin::initialize_classes() {
```

Comment:
I'm not fan of this one.
Can't we have a way to make this happens in the concerned classes (still in a static context) ?
With this design we have to modify this place each time we add a new class.

Reply:
We can't really do that. Because we have to wait for the JVM to be started anyway before initialising the classes and doing some lazy init inside the constructor would be going back to the design I removed with that PR.

Reply:
For posterity I sum up discord discussions:  
We don't have better solution to put this code to concerned classes.  
Local static init in constructor with `const char*` as template argument for name of `JavaInstanceWrapper` would have done the job, but it involves a lock and so would be unefficient for `KtObject`.